paddle/fluid/operators/mkldnn/range_mkldnn_op.cc

```diff
@@ -0,0 +1,122 @@
+/* Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/fluid/platform/mkldnn_reuse.h"
+
+namespace paddle {
+namespace operators {
+
+using framework::Tensor;
+
+template <typename T>
+class RangeMKLDNNHandler
+    : public platform::MKLDNNHandlerNoCachingT<T, dnnl::eltwise_forward,
+                                               dnnl::eltwise_backward> {
+ public:
+  RangeMKLDNNHandler(const T step, const int tensor_size,
+                     const dnnl::engine engine, platform::Place cpu_place)
```

Comment:
I know this is extremely minor and most mkldnn_reuse.h functions don't do it anyway, but I think non-primitive type arguments should always be passed by reference if possible
```suggestion
                     const dnnl::engine& engine, platform::Place& cpu_place)
```